### ISSUE TITLE
Indexer Internal_transaction fetcher stops indexing

### ISSUE THREAD

**Issue by ihsanmokhlisse** [blockscout/blockscout#5950]
after migrating blockscout into another environment (from EC2 instance to EKS), the explorer indexer can no fetch the internal_transactions 

### Environment

* Elixir & Erlang/OTP versions (`elixir -version`): Elixir 1.12.2
* Operating System: Alpine Linux 3.15
* Blockscout Version/branch: 4.1.1

### Steps to reproduce

Run BlockScout and wait

### Expected behaviour

internal transactions sync completes

### Actual behaviour

Not being able to sync new internal transactions and about of non-synced blocks increases

### Logs

```sh
 2022-08-18T16:16:34.332 fetcher=internal_transaction count=1 [error] Task #PID<0.30277.12> started from Indexer.Fetcher.InternalTransaction terminating
** (CaseClauseError) no case clause matching: {:error, [#Ecto.Changeset<action: nil, changes: %{block_number: 1233081, call_type: :staticcall, from_address_hash: %Explorer.Chain.Hash{byte_count: 20, bytes: <<93, 240, 247, 117, 137, 240, 145, 231, 171, 183, 198, 143, 11, 169, 10, 97, 81, 199, 123, 70>>}, gas: #Decimal<0>, gas_used: #Decimal<0>, index: 1, input: %Explorer.Chain.Data{bytes: <<75, 92, 66, 119, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 64, 0, ...>>}, to_address_hash: %Explorer.Chain.Hash{byte_count: 20, bytes: <<0, 0, 0, 0, 0, 0, 0, 0, 0, 99, 111, 110, 115, 111, 108, 101, 46, 108, 111, 103>>}, trace_address: [0], transaction_hash: %Explorer.Chain.Hash{byte_count: 32, bytes: <<239, 81, 9, 249, 32, 108, 68, 165, 104, 202, 208, 32, 216, 128, 188, 151, 117, 30, 215, 191, 55, 162, 80, 132, 183, 103, 223, 203, 178, 241, 102, 78>>}, transaction_index: 0, type: :call, value: #Explorer.Chain.Wei<0>}, errors: [output: {"can't be blank for successful call", [validation: :required]}], data: #Explorer.Chain.InternalTransaction<>, valid?: false>]}
    (indexer 0.1.0) lib/indexer/fetcher/internal_transaction.ex:221: Indexer.Fetcher.InternalTransaction.import_internal_transaction/2
    (indexer 0.1.0) lib/indexer/fetcher/internal_transaction.ex:126: Indexer.Fetcher.InternalTransaction.run/2
    (elixir 1.12.3) lib/task/supervised.ex:90: Task.Supervised.invoke_mfa/2
    (elixir 1.12.3) lib/task/supervised.ex:35: Task.Supervised.reply/5
    (stdlib 3.16.1) proc_lib.erl:226: :proc_lib.init_p_do_apply/3
Function: &Indexer.BufferedTask.log_run/1
    Args: [%{batch: [1233081], callback_module: Indexer.Fetcher.InternalTransaction, callback_module_state: [transport: EthereumJSONRPC.HTTP, transport_options: [http: EthereumJSONRPC.HTTP.HTTPoison, url: "https://nodeapi.test.energi.network/rpc", http_options: [recv_timeout: 60000, timeout: 1800000, hackney: [pool: :ethereum_jsonrpc]]], variant: EthereumJSONRPC.Geth], metadata: [fetcher: :internal_transaction]}]
 2022-08-18T16:16:34.339 fetcher=internal_transaction count=1 [error] Task #PID<0.30276.12> started from Indexer.Fetcher.InternalTransaction terminating
** (CaseClauseError) no case clause matching: {:error, [#Ecto.Changeset<action: nil, changes: %{block_number: 1233081, call_type: :staticcall, from_address_hash: %Explorer.Chain.Hash{byte_count: 20, bytes: <<93, 240, 247, 117, 137, 240, 145, 231, 171, 183, 198, 143, 11, 169, 10, 97, 81, 199, 123, 70>>}, gas: #Decimal<0>, gas_used: #Decimal<0>, index: 1, input: %Explorer.Chain.Data{bytes: <<75, 92, 66, 119, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 64, 0, ...>>}, to_address_hash: %Explorer.Chain.Hash{byte_count: 20, bytes: <<0, 0, 0, 0, 0, 0, 0, 0, 0, 99, 111, 110, 115, 111, 108, 101, 46, 108, 111, 103>>}, trace_address: [0], transaction_hash: %Explorer.Chain.Hash{byte_count: 32, bytes: <<239, 81, 9, 249, 32, 108, 68, 165, 104, 202, 208, 32, 216, 128, 188, 151, 117, 30, 215, 191, 55, 162, 80, 132, 183, 103, 223, 203, 178, 241, 102, 78>>}, transaction_index: 0, type: :call, value: #Explorer.Chain.Wei<0>}, errors: [output: {"can't be blank for successful call", [validation: :required]}], data: #Explorer.Chain.InternalTransaction<>, valid?: false>]}
    (indexer 0.1.0) lib/indexer/fetcher/internal_transaction.ex:221: Indexer.Fetcher.InternalTransaction.import_internal_transaction/2
    (indexer 0.1.0) lib/indexer/fetcher/internal_transaction.ex:126: Indexer.Fetcher.InternalTransaction.run/2
    (elixir 1.12.3) lib/task/supervised.ex:90: Task.Supervised.invoke_mfa/2
    (elixir 1.12.3) lib/task/supervised.ex:35: Task.Supervised.reply/5
    (stdlib 3.16.1) proc_lib.erl:226: :proc_lib.init_p_do_apply/3
Function: &Indexer.BufferedTask.log_run/1
    Args: [%{batch: [1233081], callback_module: Indexer.Fetcher.InternalTransaction, callback_module_state: [transport: EthereumJSONRPC.HTTP, transport_options: [http: EthereumJSONRPC.HTTP.HTTPoison, url: "https://nodeapi.test.energi.network/rpc", http_options: [recv_timeout: 60000, timeout: 1800000, hackney: [pool: :ethereum_jsonrpc]]], variant: EthereumJSONRPC.Geth], metadata: [fetcher: :internal_transaction]}]
 2022-08-18T16:16:34.340 fetcher=internal_transaction count=1 [error] Task #PID<0.30278.12> started from Indexer.Fetcher.InternalTransaction terminating
** (CaseClauseError) no case clause matching: {:error, [#Ecto.Changeset<action: nil, changes: %{block_number: 1233081, call_type: :staticcall, from_address_hash: %Explorer.Chain.Hash{byte_count: 20, bytes: <<93, 240, 247, 117, 137, 240, 145, 231, 171, 183, 198, 143, 11, 169, 10, 97, 81, 199, 123, 70>>}, gas: #Decimal<0>, gas_used: #Decimal<0>, index: 1, input: %Explorer.Chain.Data{bytes: <<75, 92, 66, 119, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 64, 0, ...>>}, to_address_hash: %Explorer.Chain.Hash{byte_count: 20, bytes: <<0, 0, 0, 0, 0, 0, 0, 0, 0, 99, 111, 110, 115, 111, 108, 101, 46, 108, 111, 103>>}, trace_address: [0], transaction_hash: %Explorer.Chain.Hash{byte_count: 32, bytes: <<239, 81, 9, 249, 32, 108, 68, 165, 104, 202, 208, 32, 216, 128, 188, 151, 117, 30, 215, 191, 55, 162, 80, 132, 183, 103, 223, 203, 178, 241, 102, 78>>}, transaction_index: 0, type: :call, value: #Explorer.Chain.Wei<0>}, errors: [output: {"can't be blank for successful call", [validation: :required]}], data: #Explorer.Chain.InternalTransaction<>, valid?: false>]}
    (indexer 0.1.0) lib/indexer/fetcher/internal_transaction.ex:221: Indexer.Fetcher.InternalTransaction.import_internal_transaction/2
    (indexer 0.1.0) lib/indexer/fetcher/internal_transaction.ex:126: Indexer.Fetcher.InternalTransaction.run/2
    (elixir 1.12.3) lib/task/supervised.ex:90: Task.Supervised.invoke_mfa/2
    (elixir 1.12.3) lib/task/supervised.ex:35: Task.Supervised.reply/5
    (stdlib 3.16.1) proc_lib.erl:226: :proc_lib.init_p_do_apply/3
Function: &Indexer.BufferedTask.log_run/1
    Args: [%{batch: [1233081], callback_module: Indexer.Fetcher.InternalTransaction, callback_module_state: [transport: EthereumJSONRPC.HTTP, transport_options: [http: EthereumJSONRPC.HTTP.HTTPoison, url: "https://nodeapi.test.energi.network/rpc", http_options: [recv_timeout: 60000, timeout: 1800000, hackney: [pool: :ethereum_jsonrpc]]], variant: EthereumJSONRPC.Geth], metadata: [fetcher: :internal_transaction]}]
 ```

**Comment by vbaranov** [comment]
@ihsanmokhlisse  I suppose you use Geth fro archive node, right? Which version of Geth is used?

**Comment by vbaranov** [comment]
The issue is that the response from `debug_traceTransaction` method doesn't contain `output` field, which Blockscout expects to be returned from the node. For instance, what is the response from your archive node:
```
curl -H "Content-type:application/json" —data '{"jsonrpc":"2.0","method":"debug_traceTransaction","params":["0xef5109f9206c44a568cad020d880bc97751ed7bf37a25084b767dfcbb2f1664e", {"tracer": "callTracer"}],"id":1}'
```

?

**Comment by ihsanmokhlisse** [comment]
Thank you, @vbaranov, for your quick response, below is the requested output: 
```json

{
    "jsonrpc": "2.0",
    "id": 1,
    "result": {
        "type": "CREATE",
        "from": "0x4f3c54f88a829ca5970147689010326eb12fd7b3",
        "to": "0x5df0f77589f091e7abb7c68f0ba90a6151c77b46",
        "value": "0x0",
        "gas": "0x60412",
        "gasUsed": "0x60412",
        "input": "0x60806040523480156200001157600080fd5b5060405162000c3238038062000c32833981810160405281019062000037919062000278565b6200006760405180606001604052806022815260200162000c1060229139826200008760201b620001ce1760201c565b80600090805190602001906200007f92919062000156565b5050620004c5565b620001298282604051602401620000a0929190620002fe565b6040516020818303038152906040527f4b5c4277000000000000000000000000000000000000000000000000000000007bffffffffffffffffffffffffffffffffffffffffffffffffffffffff19166020820180517bffffffffffffffffffffffffffffffffffffffffffffffffffffffff83818316178352505050506200012d60201b60201c565b5050565b60008151905060006a636f6e736f6c652e6c6f679050602083016000808483855afa5050505050565b8280546200016490620003ea565b90600052602060002090601f016020900481019282620001885760008555620001d4565b82601f10620001a357805160ff1916838001178555620001d4565b82800160010185558215620001d4579182015b82811115620001d3578251825591602001919060010190620001b6565b5b509050620001e39190620001e7565b5090565b5b8082111562000202576000816000905550600101620001e8565b5090565b60006200021d620002178462000362565b62000339565b9050828152602081018484840111156200023657600080fd5b62000243848285620003b4565b509392505050565b600082601f8301126200025d57600080fd5b81516200026f84826020860162000206565b91505092915050565b6000602082840312156200028b57600080fd5b600082015167ffffffffffffffff811115620002a657600080fd5b620002b4848285016200024b565b91505092915050565b6000620002ca8262000398565b620002d68185620003a3565b9350620002e8818560208601620003b4565b620002f381620004b4565b840191505092915050565b600060408201905081810360008301526200031a8185620002bd565b90508181036020830152620003308184620002bd565b90509392505050565b60006200034562000358565b905062000353828262000420565b919050565b6000604051905090565b600067ffffffffffffffff82111562000380576200037f62000485565b5b6200038b82620004b4565b9050602081019050919050565b600081519050919050565b600082825260208201905092915050565b60005b83811015620003d4578082015181840152602081019050620003b7565b83811115620003e4576000848401525b50505050565b600060028204905060018216806200040357607f821691505b602082108114156200041a576200041962000456565b5b50919050565b6200042b82620004b4565b810181811067ffffffffffffffff821117156200044d576200044c62000485565b5b80604052505050565b7f4e487b7100000000000000000000000000000000000000000000000000000000600052602260045260246000fd5b7f4e487b7100000000000000000000000000000000000000000000000000000000600052604160045260246000fd5b6000601f19601f8301169050919050565b61073b80620004d56000396000f3fe608060405234801561001057600080fd5b50600436106100365760003560e01c8063a41368621461003b578063cfae321714610057575b600080fd5b6100556004803603810190610050919061043d565b610075565b005b61005f61013c565b60405161006c91906104b7565b60405180910390f35b6101226040518060600160405280602381526020016106e3602391396000805461009e90610610565b80601f01602080910402602001604051908101604052809291908181526020018280546100ca90610610565b80156101175780601f106100ec57610100808354040283529160200191610117565b820191906000526020600020905b8154815290600101906020018083116100fa57829003601f168201915b50505050508361026a565b8060009080519060200190610138929190610332565b5050565b60606000805461014b90610610565b80601f016020809104026020016040519081016040528092919081815260200182805461017790610610565b80156101c45780601f10610199576101008083540402835291602001916101c4565b820191906000526020600020905b8154815290600101906020018083116101a757829003601f168201915b5050505050905090565b61026682826040516024016101e49291906104d9565b6040516020818303038152906040527f4b5c4277000000000000000000000000000000000000000000000000000000007bffffffffffffffffffffffffffffffffffffffffffffffffffffffff19166020820180517bffffffffffffffffffffffffffffffffffffffffffffffffffffffff8381831617835250505050610309565b5050565b61030483838360405160240161028293929190610510565b6040516020818303038152906040527f2ced7cef000000000000000000000000000000000000000000000000000000007bffffffffffffffffffffffffffffffffffffffffffffffffffffffff19166020820180517bffffffffffffffffffffffffffffffffffffffffffffffffffffffff8381831617835250505050610309565b505050565b60008151905060006a636f6e736f6c652e6c6f679050602083016000808483855afa5050505050565b82805461033e90610610565b90600052602060002090601f01602090048101928261036057600085556103a7565b82601f1061037957805160ff19168380011785556103a7565b828001600101855582156103a7579182015b828111156103a657825182559160200191906001019061038b565b5b5090506103b491906103b8565b5090565b5b808211156103d15760008160009055506001016103b9565b5090565b60006103e86103e384610581565b61055c565b90508281526020810184848401111561040057600080fd5b61040b8482856105ce565b509392505050565b600082601f83011261042457600080fd5b81356104348482602086016103d5565b91505092915050565b60006020828403121561044f57600080fd5b600082013567ffffffffffffffff81111561046957600080fd5b61047584828501610413565b91505092915050565b6000610489826105b2565b61049381856105bd565b93506104a38185602086016105dd565b6104ac816106d1565b840191505092915050565b600060208201905081810360008301526104d1818461047e565b905092915050565b600060408201905081810360008301526104f3818561047e565b90508181036020830152610507818461047e565b90509392505050565b6000606082019050818103600083015261052a818661047e565b9050818103602083015261053e818561047e565b90508181036040830152610552818461047e565b9050949350505050565b6000610566610577565b90506105728282610642565b919050565b6000604051905090565b600067ffffffffffffffff82111561059c5761059b6106a2565b5b6105a5826106d1565b9050602081019050919050565b600081519050919050565b600082825260208201905092915050565b82818337600083830152505050565b60005b838110156105fb5780820151818401526020810190506105e0565b8381111561060a576000848401525b50505050565b6000600282049050600182168061062857607f821691505b6020821081141561063c5761063b610673565b5b50919050565b61064b826106d1565b810181811067ffffffffffffffff8211171561066a576106696106a2565b5b80604052505050565b7f4e487b7100000000000000000000000000000000000000000000000000000000600052602260045260246000fd5b7f4e487b7100000000000000000000000000000000000000000000000000000000600052604160045260246000fd5b6000601f19601f830116905091905056fe4368616e67696e67206772656574696e672066726f6d202725732720746f2027257327a264697066735822122062b06e5bdee39e73f7ae7ef8606fe9f23da851629e4e297316ce7747f5074b1964736f6c634300080400334465706c6f79696e67206120477265657465722077697468206772656574696e673a0000000000000000000000000000000000000000000000000000000000000020000000000000000000000000000000000000000000000000000000000000000f48656c6c6f2c2048617264686174210000000000000000000000000000000000",
        "output": "0x608060405234801561001057600080fd5b50600436106100365760003560e01c8063a41368621461003b578063cfae321714610057575b600080fd5b6100556004803603810190610050919061043d565b610075565b005b61005f61013c565b60405161006c91906104b7565b60405180910390f35b6101226040518060600160405280602381526020016106e3602391396000805461009e90610610565b80601f01602080910402602001604051908101604052809291908181526020018280546100ca90610610565b80156101175780601f106100ec57610100808354040283529160200191610117565b820191906000526020600020905b8154815290600101906020018083116100fa57829003601f168201915b50505050508361026a565b8060009080519060200190610138929190610332565b5050565b60606000805461014b90610610565b80601f016020809104026020016040519081016040528092919081815260200182805461017790610610565b80156101c45780601f10610199576101008083540402835291602001916101c4565b820191906000526020600020905b8154815290600101906020018083116101a757829003601f168201915b5050505050905090565b61026682826040516024016101e49291906104d9565b6040516020818303038152906040527f4b5c4277000000000000000000000000000000000000000000000000000000007bffffffffffffffffffffffffffffffffffffffffffffffffffffffff19166020820180517bffffffffffffffffffffffffffffffffffffffffffffffffffffffff8381831617835250505050610309565b5050565b61030483838360405160240161028293929190610510565b6040516020818303038152906040527f2ced7cef000000000000000000000000000000000000000000000000000000007bffffffffffffffffffffffffffffffffffffffffffffffffffffffff19166020820180517bffffffffffffffffffffffffffffffffffffffffffffffffffffffff8381831617835250505050610309565b505050565b60008151905060006a636f6e736f6c652e6c6f679050602083016000808483855afa5050505050565b82805461033e90610610565b90600052602060002090601f01602090048101928261036057600085556103a7565b82601f1061037957805160ff19168380011785556103a7565b828001600101855582156103a7579182015b828111156103a657825182559160200191906001019061038b565b5b5090506103b491906103b8565b5090565b5b808211156103d15760008160009055506001016103b9565b5090565b60006103e86103e384610581565b61055c565b90508281526020810184848401111561040057600080fd5b61040b8482856105ce565b509392505050565b600082601f83011261042457600080fd5b81356104348482602086016103d5565b91505092915050565b60006020828403121561044f57600080fd5b600082013567ffffffffffffffff81111561046957600080fd5b61047584828501610413565b91505092915050565b6000610489826105b2565b61049381856105bd565b93506104a38185602086016105dd565b6104ac816106d1565b840191505092915050565b600060208201905081810360008301526104d1818461047e565b905092915050565b600060408201905081810360008301526104f3818561047e565b90508181036020830152610507818461047e565b90509392505050565b6000606082019050818103600083015261052a818661047e565b9050818103602083015261053e818561047e565b90508181036040830152610552818461047e565b9050949350505050565b6000610566610577565b90506105728282610642565b919050565b6000604051905090565b600067ffffffffffffffff82111561059c5761059b6106a2565b5b6105a5826106d1565b9050602081019050919050565b600081519050919050565b600082825260208201905092915050565b82818337600083830152505050565b60005b838110156105fb5780820151818401526020810190506105e0565b8381111561060a576000848401525b50505050565b6000600282049050600182168061062857607f821691505b6020821081141561063c5761063b610673565b5b50919050565b61064b826106d1565b810181811067ffffffffffffffff8211171561066a576106696106a2565b5b80604052505050565b7f4e487b7100000000000000000000000000000000000000000000000000000000600052602260045260246000fd5b7f4e487b7100000000000000000000000000000000000000000000000000000000600052604160045260246000fd5b6000601f19601f830116905091905056fe4368616e67696e67206772656574696e672066726f6d202725732720746f2027257327a264697066735822122062b06e5bdee39e73f7ae7ef8606fe9f23da851629e4e297316ce7747f5074b1964736f6c63430008040033",
        "time": "2.473696ms",
        "calls": [
            {
                "type": "STATICCALL",
                "from": "0x5df0f77589f091e7abb7c68f0ba90a6151c77b46",
                "to": "0x000000000000000000636f6e736f6c652e6c6f67",
                "input": "0x4b5c4277000000000000000000000000000000000000000000000000000000000000004000000000000000000000000000000000000000000000000000000000000000a000000000000000000000000000000000000000000000000000000000000000224465706c6f79696e67206120477265657465722077697468206772656574696e673a000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000f48656c6c6f2c2048617264686174210000000000000000000000000000000000"
            }
        ]
    }
}

```

**Comment by ihsanmokhlisse** [comment]
For geth we use another binary which is compatible with geth

```sh 
Energi3 
Version: 3.1.1-stable 
Architecture: arm64 
Protocol Versions: [70] 
Network Id: 39797 
Go Version: go1.15.8 
Operating System: linux 
GOPATH= 
GOROOT=/usr/local/go
```

**Comment by vbaranov** [comment]
> For geth we use another binary which is compatible with geth

It is compatible with which version of Geth?

As you can see:
```
"calls": [
            {
                "type": "STATICCALL",
                "from": "0x5df0f77589f091e7abb7c68f0ba90a6151c77b46",
                "to": "0x000000000000000000636f6e736f6c652e6c6f67",
                "input": "0x4b5c4277000000000000000000000000000000000000000000000000000000000000004000000000000000000000000000000000000000000000000000000000000000a000000000000000000000000000000000000000000000000000000000000000224465706c6f79696e67206120477265657465722077697468206772656574696e673a000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000f48656c6c6f2c2048617264686174210000000000000000000000000000000000"
            }
        ]
```

doesn't contain `output` parameter, which I suppose should be there for successfull transaction.

**Comment by vbaranov** [comment]
Lack of this param in response causes the error in Blockscout, which you posted initially.

**Comment by uniwswap** [comment]
any update?

**Comment by uniwswap** [comment]
@ihsanmokhlisse did you resolve it?

**Comment by uniwswap** [comment]
@vbaranov  can we discuss this issue here?

**Comment by ihsanmokhlisse** [comment]
@uniwswap, yes I did solve this issue, in my side, it's not a clean solution , but it helps to achieve what I need
my actions:
- put the explorer indexer in a loop and force the explorer indexer to restart each 6 hours
- use an empty value for this ENV VAR "ETHEREUM_JSONRPC_WS_URL"

only with this i could catch and sync my internal Transactions

**Comment by uniwswap** [comment]
how can we achieve `put the explorer indexer in a loop and force the explorer indexer to restart each 6 hours` ? @ihsanmokhlisse

**Comment by ihsanmokhlisse** [comment]
@uniwswap i'm using Kubernetes Workload to do that ;) , because I'm deploying the blockscout in Kubernetes environment

**Comment by uniwswap** [comment]
@ihsanmokhlisse  but if flag `INDEXER_DISABLE_INTERNAL_TRANSACTIONS_FETCHER` is `true` then on header part this message `Indexing Internal Transactions - We're indexing this chain right now. Some of the counts may be inaccurate.` will always be there right ??

Still I already implemented your solution, let's see in few hrs how much it is synced! 
Do you have any idea how can I check it is progressing or stuck?
@ihsanmokhlisse 
I am really thankful to your reply! Really appreciate it

**Comment by ihsanmokhlisse** [comment]
@uniwswap This is the config that I use to force indexer to be restarted : (by the way, i'm splitting the blockscout functionality into 2 components -- webapp and indexer)
```yaml 
apiVersion: v1
items:
- apiVersion: batch/v1
  kind: CronJob
  metadata:
    annotations:
      meta.helm.sh/release-name: energi-explorer-testnet
      meta.helm.sh/release-namespace: energi-explorer-testnet
    generation: 1
    labels:
      app.kubernetes.io/managed-by: Helm
    name: restart-indexer
    namespace: energi-explorer-testnet
  spec:
    concurrencyPolicy: Forbid
    failedJobsHistoryLimit: 1
    jobTemplate:
      metadata:
        creationTimestamp: null
      spec:
        activeDeadlineSeconds: 600
        backoffLimit: 2
        template:
          metadata:
            creationTimestamp: null
          spec:
            containers:
            - command:
              - kubectl
              - rollout
              - restart
              - deployment
              - energi-explorer-testnet-indexer
              image: bitnami/kubectl
              imagePullPolicy: Always
              name: kubectl
              resources: {}
              terminationMessagePath: /dev/termination-log
              terminationMessagePolicy: File
            dnsPolicy: ClusterFirst
            restartPolicy: Never
            schedulerName: default-scheduler
            securityContext: {}
            serviceAccount: restart-indexer
            serviceAccountName: restart-indexer
            terminationGracePeriodSeconds: 30
        ttlSecondsAfterFinished: 1
    schedule: 0 */12 * * *
    successfulJobsHistoryLimit: 3
    suspend: false
  status:
kind: List
metadata:
  resourceVersion: ""
```

**Comment by ihsanmokhlisse** [comment]
@uniwswap , from my side I check directly the DB and see if the number of skipped internal transaction is increasing or decreasing 

```sql
select count(s.i) AS missing_mn_nos FROM generate_series( 1240000, 1281272) s(i) LEFT OUTER JOIN internal_transactions ON (internal_transactions.block_number = s.i) WHERE internal_transactions.block_number IS NULL;
 missing_mn_nos
```

those numbers "( 1240000, 1281272)" represent your blocks from your blockchain

**Comment by uniwswap** [comment]
I am using systmd service so I am restarting every one hour to check it is working or not!

Let me check from my side how thing is going in database and will comeback to you if require any help

Thank you

**Comment by uniwswap** [comment]
@ihsanmokhlisse No, it is not working for me :(

**Comment by uniwswap** [comment]
> Lack of this param in response causes the error in Blockscout, which you posted initially.

what will be solution around it?

**Comment by ihsanmokhlisse** [comment]
Can you share your configuration and the env variables used ?

**Comment by uniwswap** [comment]
![image](https://user-images.githubusercontent.com/113737292/199414235-5a55c27f-e523-4773-bb82-bd4a17721ce7.png)

and Configuration for instance is 8vCPU and 32Gb RAM

@ihsanmokhlisse

**Comment by ihsanmokhlisse** [comment]
@uniwswap below is the configuration used for testnet Network for information, and also to compare is with your configuration 
```yaml
  ALLOWED_EVM_VERSIONS: petersburg,homestead,tangerineWhistle,spuriousDragon,byzantium,constantinople,istanbul
  API_RATE_LIMIT: "50"
  APPS_MENU: "false"
  BLOCK_TRANSFORMER: base
  BLOCKSCOUT_HOST: explorer.test.energi.network
  BLOCKSCOUT_PROTOCOL: http
  BLOCKSCOUT_VERSION: v4.1.1
  CHAIN_ID: "49797"
  CHAIN_SPEC_PATH: chain_spec_energi_testnet.json
  CHECK_ORIGIN: "true"
  COIN: NRG
  COINGECKO_COIN_ID: energi
  DISABLE_EXCHANGE_RATES: "false"
  DISPLAY_TOKEN_ICONS: "true"
  ECTO_USE_SSL: "false"
  EMISSION_FORMAT: MNRG
  ENABLE_TXS_STATS: "true"
  ENERGISWAP_API_URL: http://energiswap-api-testnet.energiswap-testnet.svc.cluster.local:443/v1
  ETHEREUM_JSONRPC_HTTP_URL: https://nodeapi.test.energi.network/rpc
  ETHEREUM_JSONRPC_MAXCONN: "20"
  ETHEREUM_JSONRPC_TRACE_URL: https://nodeapi.test.energi.network/rpc
  ETHEREUM_JSONRPC_TRANSPORT: http
  ETHEREUM_JSONRPC_VARIANT: geth
  ETHEREUM_JSONRPC_WS_URL: ""
  EXTERNAL_APPS: '[]'
  HISTORY_FETCH_INTERVAL: "30"
  JSON_RPC: https://nodeapi.test.energi.network/v1/jsonrpc
  LINK_TO_OTHER_EXPLORERS: "false"
  LOGO: /images/EnergiLogoLight.png
  LOGO_FOOTER: /images/EnergiLogoLight.png
  MAINTENANCE_ALERT_MESSAGE: Test Energi Explorer
  MAX_SKIPPING_DISTANCE: "2"
  MIX_ENV: prod
  NETWORK: Energi
  RELEASE_LINK: https://github.com/energicryptocurrency/energi3-blockscout/releases
  SHOW_MAINTENANCE_ALERT: "false"
  SHOW_TXS_CHART: "true"
  SUBNETWORK: Testnet
  SUPPORTED_CHAINS: '[{"url":"https://explorer.test.energi.network/", "title":"Explorer
    Testnet",}]'
  TOKEN_EXCHANGE_RATE_CACHE_PERIOD: "300"
  TOKEN_METADATA_UPDATE_INTERVAL: "3600"
  WOBSERVER_ENABLED: "false"
  ```
  also, please check if your explorer takes all the resources from the instance, if i'm not mistaking the indexer limited to 4gb or ram or something similar to this

**Comment by vbaranov** [comment]
Joined issue https://github.com/blockscout/blockscout/issues/5566

**Comment by vbaranov** [comment]
This should be fixed in [5.0.0](https://github.com/blockscout/blockscout/releases/tag/v5.0.0-beta). I am closing this issue.
